hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -38,70 +38,15 @@
     "hw/top_earlgrey/data/ip/chip_sram_ctrl_testplan.hjson",
     "hw/top_earlgrey/data/ip/chip_usbdev_testplan.hjson",
     "hw/top_earlgrey/data/ip/chip_rv_timer_testplan.hjson",
+    "hw/top_earlgrey/data/ip/chip_uart_testplan.hjson",
```

Comment:
rv_timer seems to be at the wrong spot here, maybe put uart in between sram and usbdev?

Reply:
Oops! Good point. I'd noticed that `rv_timer` was in the wrong place but apparently I can't do the alphabet, so put "uart" after "usbdev". D'oh! I've just force-pushed a version that puts things in the right place.